src/SubmissionSnsEventHandler.ts

```diff
@@ -74,6 +74,11 @@ export class SubmissionSnsEventHandler extends Construct {
     secret.grantRead(submissionLambda);
     const key = Key.fromKeyArn(this, 'sourceBucketKey', StringParameter.valueForStringParameter(this, Statics.ssmSourceKeyArn));
     key.grantDecrypt(submissionLambda);
+    submissionLambda.addToRolePolicy(new PolicyStatement({
+      effect: Effect.ALLOW,
+      actions: ['events:PutEvents'],
+      resources: [`arn:aws:events:${Stack.of(this).region}:${Stack.of(this).region}:event-bus/default`],
+    }));
```

Comment:
Zo werkt het ook, en gebruik je default cdk-logica?
```suggestion
    const bus = EventBus.fromEventBusName(this, 'defaultbus', 'default');
    bus.grantPutEventsTo(submissionLambda);
```